tutorials/3d/baked_lightmaps.rst

```diff
@@ -157,65 +173,173 @@ can be controlled from the **Bake Mode** menu in lights:
+  it is recommended to decrease the number of bounces to 1 to speed up baking.
+  Remember that your scene will be darker when decreasing the number of bounces.
+- **Use Denoiser:** If enabled, uses OpenImageDenoise to make the lightmap
+  significantly less noisy. This increases bake times and can occasionally
+  introduce artifacts, but the result is often worth it.
+- **Use Hdr:** If disabled, lightmaps are smaller on disk, but they won't be
+  able to capture any light over white (1.0). This will result in visible clipping
+  if you have bright lights in your scene. When HDR is disabled, banding may also
+  be visible in the lightmap.
+- **Use Color:** If disabled, lightmaps are smaller on disk, but the lightmap
+  won't be able to store colored lighting. When baking indirect light only, the
+  difference may be barely visible since indirect light is generally not highly
+  saturated. However, when baking both direct and indirect lighting using the
+  **All** bake mode on a light, this will turn colored lighting into grayscale
+  lighting. This can be disabled together with HDR to get the smallest possible
+  lightmap file at a given resolution.
+- **Bias:** The offset value to use for shadows in 3D units. You generally don't
+  need to change this value, except if you run into issues with light bleeding or
+  dark spots in your lightmap after baking. This setting does not affect real-time
+  shadows casted on baked surfaces.
+- **Default Texels Per Unit:** For meshes that do not specify their own lightmap
+  texel density, this will be used as the value. Higher values result in
+  *lower-resolution* lightmaps, which result in faster bake times and lower file
+  sizes at the cost of blurrier indirect lighting and shadows.
+
+Atlas
+^^^^^
+
+- **Generate:** If enabled, a texture atlas will be generated for the lightmap.
+  This results in more efficient rendering, but is only compatible with the
+  GLES3 rendering backend. Disable this setting if your project is allowed to
+  fall back to GLES2. (This is not the case by default and must be enabled in
+  the Project Settings.) *This setting is ignored when the project is configured
+  to use GLES2 by default.*
+- **Max Size:** The maximum size of the atlas in pixels. Higher values result
+  in a more efficient atlas, but are less compatible with old/low-end hardware.
+  If in doubt, leave this setting on its default value (4096).
+
+Capture
+^^^^^^^
+
+- **Enabled:** This enables probe capture so that dynamic objects can *receive* indirect lighting.
+  Regardless of this setting's value, dynamic objects will not be able to
+  *contribute* indirect lighting to the scene. This is a limitation of lightmaps.
+- **Cell Size:** The distance between lightmap probes in 3D units. Higher values
+  result in more sparse probe placement, which decreases bake times and file
+  size at the cost of lower lighting accuracy for dynamic objects.
```

Comment:
Capture still uses the old voxel octree, so I would use "voxels" instead of "probes" here, otherwise it may lead to cinfusion.